mods/default/chests.lua

```diff
@@ -3,16 +3,29 @@ default.chest = {}
 -- support for MT game translation.
 local S = default.get_translator
 
-function default.chest.get_chest_formspec(pos)
+function default.chest.get_chest_formspec(pos, size)
 	local spos = pos.x .. "," .. pos.y .. "," .. pos.z
-	local formspec =
-		"size[8,9]" ..
-		"list[nodemeta:" .. spos .. ";main;0,0.3;8,4;]" ..
-		"list[current_player;main;0,4.85;8,1;]" ..
-		"list[current_player;main;0,6.08;8,3;8]" ..
-		"listring[nodemeta:" .. spos .. ";main]" ..
-		"listring[current_player;main]" ..
-		default.get_hotbar_bg(0,4.85)
+	-- Find widest and highest possible combination
+	local fs_size = {
+		x = math.max(size.x, 8),
```

Comment:
So what if `size.x > 8`? Please include an assertion if this is not supported.

Reply:
I don't know. It's been so long that I've forgotten most of this code haha

Reply:
Okay, I tried `math.min` instead; works as we need it to.